### ISSUE TITLE
Přehled kampaně

### ISSUE THREAD

**Issue by VeronikaRuliskova** [TOPefekt/BulkGate-issues#117]
v přehledu kampaně upravit graficky:
![image](https://user-images.githubusercontent.com/40599364/104308005-44416800-54d0-11eb-9857-48b4ecb65a5d.png)

rozhodně bych nedala černou to waiting to be sent a větším než outbox. 

Při prvním pohledu to chytne oko a budí to dojem, že kampaň je stále posílaná - zmatkař začne okamžitě klikat a hledat proč, kde jak..

Prioritně by měl být přehled o tom, že nemusíš přemýšlet, jak kdo co myslel

Tzn. dle mýho názoru - zmenšit to tak, aby byl outbox větší a dát tomu jinou barvu, méně výraznou než tu, co použiješ pro outbox

**Comment by LiborD** [comment]
Co něco takového? @VeronikaRuliskova @lukaspijak 
![image](https://user-images.githubusercontent.com/36260368/104443561-e592f180-5596-11eb-8acd-2278c469058e.png)

**Comment by VeronikaRuliskova** [comment]
Ano, Liborku, to je přesně ono, teď je to mnohem pochopitelnější. Děkuji

**Comment by lukaspijak** [comment]
Krasny no a kde je počet zpráv? To je to první co bude useri hledat...

**Comment by LiborD** [comment]
> Krasny no a kde je počet zpráv? To je to první co bude useri hledat...

ted je tam taky nevidim, dal sem tam %, protoze sem tam videl %...ale tak to je jedno, dej si tam cislo misto %

**Comment by lukaspijak** [comment]
A kde dám ty procenta?

**Comment by LiborD** [comment]
> A kde dám ty procenta?

tak tam dej toto třeba...nebo nejakou kombinaci...
![image](https://user-images.githubusercontent.com/36260368/104464473-40d2dd00-55b3-11eb-8d38-00f935b072ce.png)

**Comment by lukaspijak** [comment]
No tohle je hroznej zmatek a krok zpět... vubec to nereflektuje to co to má dělat...

**Comment by LiborD** [comment]
nevim, proč...máš tam obě čísla, která potřebuješ...ve své podstatě máš aj ty linky, které znázorňují, kde seš..můžeš dát do campaign statusu třeba procenta...ale jako souhlasím s verčou, protože sem taky chvíli přemýšlel, jak to vůbec funguje, co ta informace měla podat..tady jasně vidíš, kolik bude odeslaných a kolik bylo odeslaných a že to je Status kampaně..

ale fajn, možná tomu vůbec nerozumím, co to má podat, ale v tom případě to spíš usera mate?
